src/components/about/ContactCard.js

```diff
@@ -101,56 +96,45 @@ const ModalCard = ({ imageUrl, imageAlt, modalTitle, title, blurb }) => {
         as="a"
         href="#"
         ref={focusRef}
-        maxW={[null, '100%', '342px']}
+        margin="5% auto"
+        maxW={['279px', '342px']}
         maxH="322px"
         direction="column"
         onClick={onOpen}
+        marginBottom={margin}
       >
-        <CardContent
-          title={title}
-          blurb={blurb}
-          imageUrl={imageUrl}
-          imageAlt={imageAlt}
-        />
+        <CardContent title={title} blurb={blurb} publicId={publicId} />
       </Flex>
       <ModalForm isOpen={isOpen} title={modalTitle} onClose={onClose} />
     </>
   );
 };
 
-const MailtoCard = ({ imageUrl, imageAlt, email, title, blurb }) => (
+const MailtoCard = ({ publicId, email, title, blurb }) => (
   <Flex
     as="a"
     href={`mailto:${email}`}
     isExternal
-    maxW={[null, '100%', '342px']}
+    margin="5% auto"
```

Comment:
Same margin comment above here as well :)

Reply:
just so I know, if a rem is lets say 1.6256, is the exact number best practice?

Reply:
@juanri0s I usually use a tool like https://type-scale.com/ to set a nice logical stepping for font and spacing values. When a design doesn't 100% adhere to that stepping, I just choose the closest value.

This keeps things extremely consistent throughout the site, and makes it easy as you don't really have to think too much about the values!

Short answer is yes. If I am given a 47px value as spacing I'd use `2.9375rem` :)